### PR TITLE
Updated spaln, add list_spaln_tables tool

### PR DESCRIPTION
**Pull request by pvanheus**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [x] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by pvanheus** [comment]
This PR:

1) Updates spaln to version 2.4.03
2) Adds many more advanced options to the spaln wrapper including support for choosing species specific parameters
3) Adds a tool to choose the most appropriate species specific parameter file for spaln

**Comment by bgruening** [comment]
@pvanheus linting is failing afaik.

**Comment by pvanheus** [comment]
> @pvanheus linting is failing afaik.

Yup. All fixed now.

**Comment by pvanheus** [comment]
Don't merge yet, this is a work in progress.

**Comment by pvanheus** [comment]
Ready for merging now!

File: tools/spaln/list_spaln_tables.xml [hunk 1]
```diff
@@ -0,0 +1,132 @@
+<tool id="list_spaln_tables" name="List spaln parameter tables" version="0.1+galaxy0">
```
Comment (mvdbeek) [inline tools/spaln/list_spaln_tables.xml:1]: Is there a reason to not track the the spaln version here (https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#tool-versions)?

Comment (pvanheus) [reply tools/spaln/list_spaln_tables.xml:1]: Originally the tool was not strictly tied to spaln version, but might as well track it here.

**Comment by bgruening** [comment]
Thanks @pvanheus!

**Pull request merged**